package.json

```diff
@@ -9,7 +9,8 @@
     "docs:dev": "vuepress dev docs",
     "docs:build": "vuepress build docs --dest dist",
     "build": "npm run docs:build",
-    "test": "npx eslint ."
+    "test": "npx eslint .",
+    "deploy": "npm run build && bash ./deploy $1 && rm -r ./dist"
```

Comment:
@boutell When run like this, with `npm run deploy staging`, it looks like the command loses track of `staging` by the time it gets to that second step. Is there a way to keep track of that better?